plugins/swagger-apibody/src/util/ast.ts

```diff
@@ -0,0 +1,196 @@
+import * as recast from "recast";
```

Comment:
There is already a library called code-gen-utils
https://www.npmjs.com/package/@amplication/code-gen-utils

if these functions are missing, please update the library